docs/src/navigation/breadcrumbs/breadcrumbsSampleOnlyPreviousRoute.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Breadcrumbs } from 'react-cm-ui';
+import ROUTER from '../../../../src/navigation/breadcrumbs/__test__/breadcrumbsMockups';
```

Comment:
The router seems to be a long object, would it be good to include the whole mocked up object in the docs snippet? or just including it in the first sample and then referencing it for the rest of the samples?

Reply:
Ideally, we don't want a lot of "noise" in the documentation examples.
We'd want to illustrate (as close as possible) how your component would be used with a live router.
So let's try to minimize any details around "mocking" the router.

My two cents.

Reply:
Docs updated, I added a simpler router mockup.

Reply:
Yeah, I like the current samples.  You can see the route structure to inform what the sample is doing, but it's not so gross and distracting.  Good changes! 👍